REST/Documentation.json

```diff
@@ -3973,7 +4047,7 @@
           "type": "array",
           "items": {
             "type": "object",
-            "$ref": "#/definitions/CustomFieldModel"
+            "$ref": "#/definitions/JobCustomFieldModel"
```

Comment:
Is this change already merged to master?

Reply:
Yes, this is only for the website. We can change model names, because the JSON generated from model stays the same.